src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +100,13 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+  
+  /** Returns the amount of conversations currently in the database. */
+  public int getConversationAmount(){
+    if(this == null){
```

Comment:
I think this check is unnecessary as you can assume that they're calling this method on an instance that is obtained/constructed via getInstance